lib/sandbox/sandbox_linux.go

```diff
@@ -75,14 +74,17 @@ func (netns *NetNs) symlinkCreate(name string) error {
 	return nil
 }
 
+// Path returns the path of the network namespace handle
```

Comment:
From [here][1]:

> Notice this comment is a complete sentence that begins with the name of the element it describes.

So I think you want trailing periods here and below.

[1]: https://blog.golang.org/godoc-documenting-go-code